website/content/docs/register/service/aws.mdx

```diff
@@ -30,106 +34,165 @@ curl -L -o $FILENAME $DL_URL 2&>/dev/null && \
 
 ```shell-session
-$ consul-aws -aws-namespace-id ns-hjrgt3bapp7phzff -to-aws -consul-service-prefix my_hcp_consul_cluster_
+$ consul-aws \
+     -aws-namespace-id ns-hjrgt3bapp7phzff \
+     -to-consul\
+     -aws-service-prefix my_aws_cloudmap_namespace_
 ```
 
-After the command starts, `consul-aws` begins importing services from Consul to AWS CloudMap. A service in Consul with
-the name`web` identifies as `my_hcp_consul_cluster_web` in AWS CloudMap.
+Consul sets the following properties for services it syncs from AWS Cloud Map to Consul.
 
-Consul sets the following properties for services synced _to_ AWS CloudMap:
+|Property|Value|
+|---|---|
+|Tag|aws|
+|Meta-Data|includes `aws` as the source set, the `aws-id`, the `aws-namespace` and custom attributes the instance uses in AWS Cloud Map|
+|Node|the node name is `consul-aws`|
 
-- Description: "Imported from Consul"
-- Record types: A and SRV
-- DNS routing policy: Multivalue answer routing
+## Sync two-way between Consul and AWS Cloud Map
 
-## Sync AWS CloudMap namespaces one-way to Consul
+To enable a two-way or bidirectional sync between Consul and an AWS Cloud Map namespace, add both the `-to-consul` and `-to-aws` flags.
 
-To sync services from an AWS CloudMap namespace to Consul, the `-to-consul` flag enables syncing from AWS to Consul. If
-you are using HCP Consul Dedicated or hosting services across multiple cloud providers, one strategy for prefixing is adding
-`aws` somewhere in the prefix. This lets you quickly differentiate services imported with this tool, and other
- hosted Consul services. The prefix is fully customizable to your choosing.
+The `-aws-namespace-id` flag is required. but the `-aws-service-prefix` and `-consul-service-prefix` flags are optional.
 
 ```shell-session
-$ consul-aws -aws-namespace-id ns-hjrgt3bapp7phzff -to-consul -aws-service-prefix my_aws_account_
+$ consul-aws \
+     -aws-namespace-id ns-hjrgt3bapp7phzff \
+     -to-consul \
+     -aws-service-prefix my_aws_cloudmap_namespace_ \
+     -to-aws \
+     -consul-service-prefix my_hcp_consul_cluster_
 ```
 
-After the command starts, `consul-aws` begins importing services from the CloudMap namespace to Consul. A
-service in AWS CloudMap with the name `redis` identifies as `my_cloudmap_namespace_redis` in Consul.
+## Next steps
 
-Consul sets the following properties for services synced _from_ AWS CloudMap:
+After setting up a sync between Consul and AWS Cloud Map, you can further manage your services with the following resources.
 
-- Tag: aws
-- Meta-Data: includes `aws` as the source set, the `aws-id`, the `aws-namespace` and custom attributes the instance uses in AWS CloudMap
-- Node: the node name is `consul-aws`
+- [Connect services with service mesh](/consul/docs/connect)
+- [Manage application traffic](/consul/docs/manage-traffic)
+- [Observe service mesh traffic](/consul/docs/observe)
 
-## Bidirectional syncing
+## `consul-aws` configuration reference
```

Comment:
Wasn't sure if this should be removed since I think we generally don't want to document other tools but given that 1) it's still a hashicorp tool and 2) there's no CLI reference even on the official tool page, I figured it might be good to include it here.